src/c/user-settings.js

```diff
@@ -0,0 +1,435 @@
+                                                        ' Senha atual'
+                                                    ),
+                                                    m('input.password.optional.w-input.text-field.w-input.text-field.positive[id=\'user_current_password\'][name=\'user[current_password]\'][type=\'password\']', {
+                                                      value: fields.current_password(),
+                                                      onchange: m.withAttr('value', fields.current_password)
+                                                    })
+                                                ]
+                                            ),
+                                            m('.w-col.w-col-6',
+                                                [
+                                                    m('label.field-label.fontweight-semibold',
+                                                        ' Nova senha'
+                                                    ),
+                                                    m('input.password.optional.w-input.text-field.w-input.text-field.positive[id=\'user_password\'][name=\'user[password]\'][type=\'password\']', {
+                                                      value: fields.password(),
+                                                      onchange: m.withAttr('value', fields.password)
+                                                    })
+                                                ]
+                                            )
+                                        ]
+                                    ),
+                                    m('.divider.u-marginbottom-20'),
+                                    m('.fontweight-semibold.fontsize-smaller',
+                                        'Desativar minha conta'
+                                    ),
+                                    m('.fontsize-smallest',
+                                        'Todos os seus apoios serão convertidos em apoios anônimos, seus dados não serão mais visíveis, você sairá automaticamente do sistema e sua conta será desativada permanentemente.'
+                                    ),
+                                    m(`a.alt-link.fontsize-smaller[data-confirm=\'Você deseja desativar essa conta?\'][data-method=\'delete\'][href=\'/pt/users/${user.id}\'][rel=\'nofollow\']`,
+                                        'Desativar minha conta no Catarse'
+                                    )
+                                ]
+                            )
+                        )
+                    ),
+                    m('div',
+                        m('.w-container',
+                            m('.w-row',
+                                [
+                                    m('.w-col.w-col-4'),
+                                    m('.w-col.w-col-4',
```

Comment:
Every time we see an empty div with set width like this we can replace it for a `-push-` class. In this case, remove the empty div and add the class `.w-col-push-4` on the main div.